File: ueueo-framework/ueueo-localization/src/main/java/Abp/Localization/ExceptionHandling/AbpExceptionLocalizationOptions.cs
using System;
using System.Collections.Generic;

namespace Volo.Abp.Localization.ExceptionHandling;

public class AbpExceptionLocalizationOptions
{
    public Dictionary<String, Type> ErrorCodeNamespaceMappings;//  { get; }

    public AbpExceptionLocalizationOptions()
    {
        ErrorCodeNamespaceMappings = new Dictionary<String, Type>();
    }

    public void MapCodeNamespace(String errorCodeNamespace, Type type)
    {
        ErrorCodeNamespaceMappings[errorCodeNamespace] = type;
    }
}

File: ueueo-frameworks/ueueo-localization/src/main/java/com/ueueo/localization/exceptionhandling/AbpExceptionLocalizationOptions.java
package com.ueueo.localization.exceptionhandling;

import lombok.Data;

import java.util.HashMap;
import java.util.Map;

@Data
public class AbpExceptionLocalizationOptions {

    private Map<String, Class<?>> errorCodeNamespaceMappings;

    public AbpExceptionLocalizationOptions() {
        errorCodeNamespaceMappings = new HashMap<>();
    }

    public void mapCodeNamespace(String errorCodeNamespace, Class<?> type) {
        errorCodeNamespaceMappings.put(errorCodeNamespace, type);
    }
}
